lib/byebug/helpers/toggle.rb

```diff
@@ -26,14 +24,22 @@ def enable_disable_breakpoints(is_enable, args)
           selected_breakpoints = all_breakpoints.select do |b|
             selected_ids.include?(b.id)
           end
+          return selected_breakpoints
```

Comment:
I don't have time to work on this and don't see in the foreseeable future having the time. These changes were motivated by requests from others for better support for byebug in realgud.

The harder it is to get changes in, the less likely I am to work on this but rather on something that is more satisfying, more to my needs, and is less effort. 

I recall someone once asking Matz about working on a standard test suite using rspec or TDD, and his reply as I recall was: "I could spend my time working on the test spec or working on Ruby. Which is more fun?"